docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -398,13 +398,58 @@ saved object ID of the action.
 
 |==============================================
 
+[discrete]
+[[optional-actions-fields-bulk-update]]
+===== Optional `action` fields
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|frequency |String a|Object containing an action’s frequency:
+
+* `summary` (Boolean, required): Defines whether to send notifications as a summary of alerts or for each generated alert.
```

Comment:
So your edit is partially accurate, but I don't want to remove "notifications" because this field dictates how notifications are sent (i.e., as alert summaries or as individual notices).

```suggestion
* `summary` (Boolean, required): Defines whether to send notifications alert summaries or for individual alerts. 
```